coordinator/src/bin/coordinator.rs

```diff
@@ -192,8 +196,16 @@ async fn main() -> Result<()> {
         }
     });
 
+    let (tx_user_feed, _rx) = broadcast::channel::<NewUserMessage>(100);
     let (tx_price_feed, _rx) = broadcast::channel(100);
-    let (_handle, trading_sender) = trading::start(pool.clone(), tx_price_feed.clone());
+
+    let (_handle, notifier) = notification::start(tx_user_feed.clone());
+
+    let (_handle, trading_sender) =
+        trading::start(pool.clone(), tx_price_feed.clone(), notifier.clone());
+
+    let _handle = async_match::monitor(pool.clone(), tx_user_feed.clone(), notifier.clone());
+    let _handle = rollover::monitor(pool.clone(), tx_user_feed.clone(), notifier);
```

Comment:
Btw, I think there is not much benefit to just returning the `RemoteHandle` to "ignore" it. What you usually do with a `RemoteHandle` is store in some other struct so that when you drop that one the corresponding task is cancelled.

Reply:
> > ❓ Doesn't the fact that you shadow _handle cause the previous RemoteHandles to be dropped stopping the corresponding tasks?
> 
> Doesn't look like it. At least it worked during testing.

Confirmed it does not drop it: https://www.reddit.com/r/rust/comments/qpekne/what_actually_happens_when_ypu_shadow_a_variable/.

Reply:
> > Btw, I think there is not much benefit to just returning the RemoteHandle to "ignore" it. What you usually do with a RemoteHandle is store in some other struct so that when you drop that one the corresponding task is cancelled.
> 
> Well the reason I return it to the top level is so that the tasks live as long as the coordinator. That's why I am doing this, do you think that is not needed?

I guess because the coordinator is the whole program it isn't actually useful. It would be handy if we could remake the coordinator in case of catastrophic failure (without restarting the binary). Then you would want to cancel all the old tasks and restart them with a fresh coordinator struct.

> Happy to remove it, but I think when I did, the compiler warned me about it.

That's probably because you called `remote_handle` on the future and returned the `RemoteHandle` which is marked as `must_use`. You can obviously just spawn the future without doing any of that, but I don't mind if you wanna keep the current version.

Reply:
got ya. Since this PR is now open for quite some time. I prefer to go ahead with the current change, and adapt the `remote_handle` stuff in a follow up PR.